tests/frame/test_common.py

```diff
@@ -151,10 +152,11 @@ def test_schema(df_raw: Any) -> None:
     assert result == expected
 
 
+# TODO(Unassigned): Fix polars 1.0 compatibility
+# 313
 @pytest.mark.parametrize(
     "df_raw", [df_pandas, df_lazy, df_pandas_nullable, df_pandas_pyarrow]
 )
-# TODO(Unassigned): https://github.com/narwhals-dev/narwhals/issues/313
```

Comment:
thanks @ELC 

does it work to keep links to github issues? like this we can just ctrl+click from IDEs

Reply:
Seems it does, nice suggestion